earth_enterprise/src/fusion/tools/gee_test/README

```diff
@@ -1,58 +1,90 @@
-#
-# Copyright 2017 Google Inc.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
 
 Tool for installing Fusion and Earth Server, building tutorial databases (2d, 3d, and 3d Time Machine), and running tests against these databases.
 
 The clean is harsh so you should not use it on machines where you want to save your work.
 
 The test program does not work unless the asset root /usr/local/google/gevol_test already exists. This seems to be an issue with the clean install. If it fails, you will have to run:
 sudo /opt/google/bin/geconfigureassetroot --new --assetroot  /usr/local/google/gevol_test/assets
-# You can add source volumes:
+
+You must also add source volumes with the names "src" and "tutorial", respectively:
 /usr/local/google/gevol_test/src
 /opt/google/share/tutorials/fusion
 
 sudo /opt/google/bin/geselectassetroot --assetroot  /usr/local/google/gevol_test/assets
 
+sudo /opt/google/bin/geselectpublishroot /usr/local/google/gevol_test/published_db
```

Comment:
The publish_tutorial script didn't work for me until I created the asset root.  We should maybe replace this command with the one below.

```
sudo /opt/google/bin/geconfigurepublishroot --path=/usr/local/google/gevol_test/published_db
```